common/views/components/Pagination/Pagination.tsx

```diff
@@ -148,15 +149,17 @@ export const Pagination: FunctionComponent<Props> = ({
         </Link>
       )}
 
-      {isEnhanced && isHiddenMobile ? (
```

Comment:
Ahh I think this is what is causing the UI change?

Reply:
yes, the jump to page should always be visible where the pagination component is used